Boss/Mod/ConnectFinderByHardcode.cpp

```diff
@@ -16,39 +16,32 @@ namespace {
-    , "032434517e28f7b51665a525d5e11fa493bd4e30a59883b8156c2be4085f4aaf70@46.6.9.152:9735"
-    , "029dd81c32e3ea839b4b2f8edcc9bea227c40fa2c6509878034401a8ce2a8bac05@176.36.37.21:9735"
-    , "0215e9649c6a14ca64109418a5b17b0ddefd446e8256486a0c692e1fd8ec72e316@200.75.4.254:9735"
-    , "03c5528c628681aa17ab9e117aa3ee6f06c750dfb17df758ecabcd68f1567ad8c1@82.16.240.248:9735"
-    , "035fed4182fbd0725264f8a0018cabb6b25514dd231291162ac8dd63afb278e9e8@68.183.121.169:9735"
-    , "03b323009613777edc4e8024c6726758396bc026da99545971164a6baad31dfc69@18.176.200.244:9735"
-    , "028c8c9b0b588d00afabe905799aee0225a9315cdb63d7646b7ff7cf02fe4bf643@197.155.6.43:9735"
-    , "0326f1d06748f019ecd98484eb23ba7367b1361e0ee6e38234d73afcc976dfdb3d@190.85.201.37:9735"
-    , "03f3f3f9ea1e6e3df8108a3f11031f0fba8e494535c5a45fb83bdc95a89fbe18b7@189.212.121.74:9735"
-    , "035e3aa81503b67d61589270c043bcf6dcf083e40bc8630eeeeb08f85570be0fc9@185.123.207.109:9735"
-    , "02309f3556d51e9d855594762dcf68cb283fbfd7e48618b6cecb8cd90e2e34d8ad@79.62.233.36:9735"
-    , "0321e0fedfda0722c3eaa866cea8a23540dae4dfbaf247785f7fe111b0ea7a53ec@49.0.65.207:9735"
-    , "027e70d8859e090ed975b11cfa02109221482954a254dff185e4142188eb66518b@47.75.125.192:9735"
-    , "0237fefbe8626bf888de0cad8c73630e32746a22a2c4faa91c1d9877a3826e1174@13.48.89.186:9735"
+  , { "03e2408a49f07d2f4083a47344138ef89e7617e63919202c92aa8d49b574a560ae@65.108.246.14:19736"
+    , "035e4ff418fc8b5554c5d9eea66396c227bd429a3251c8cbc711002ba215bfc226@170.75.163.209:9735"
+    , "03abf6f44c355dec0d5aa155bdbdd6e0c8fefe318eff402de65c6eb2e1be55dc3e@3.132.230.42:9735"
+    , "03b65c19de2da9d35895b37e6fa263cefbf8d184e6b61c0713747ebf12409d219f@164.90.241.111:9735"
+    , "02d96eadea3d780104449aca5c93461ce67c1564e2e1d73225fa67dd3b997a6018@45.86.229.190:9736"
+    , "031f2669adab71548fad4432277a0d90233e3bc07ac29cfb0b3e01bd3fb26cb9fa@44.242.118.94:9735"
+    , "03cde60a6323f7122d5178255766e38114b4722ede08f7c9e0c5df9b912cc201d6@34.65.85.39:9745"
+    , "02f1a8c87607f415c8f22c00593002775941dea48869ce23096af27b0cfdcc0b69@52.13.118.208:9735"
+    , "0294ac3e099def03c12a37e30fe5364b1223fd60069869142ef96580c8439c2e0a@8.210.134.135:26658",
+    }
+  }
+  , { Boss::Msg::Network_Testnet
+    ,  { "039fbf81305e417c2acbe3b96a4929ce40b7f3537720199db9dc010b1be3c28553:167.99.21.222:19735"
+       , "030b686a163aa2bba03cebb8bab7778fac251536498141df0a436d688352d426f6:65.108.246.14:19735"
+       , "03781444afe27babf724304c25370f195d9d955065d7894539ed7db5fe596badb8@138.197.57.50:19735"
+       , "021713d5331898c206b57c4f7d40635079de9a97d97782646f31dac18a53f2d979@66.241.124.221:9735"
+       , "03e84a109cd70e57864274932fc87c5e6434c59ebb8e6e7d28532219ba38f7f6df@139.144.22.237:9735"
+       , "03f060953bef5b777dc77e44afa3859d022fc1a77c55138deb232ad7255e869c00@212.46.38.66:9735"
+       , "036497fee1a0b963a63f0b4374819d9016fcca9c09dbef4cb427a2d0146649b025@44.234.48.207:9735",
+    }
+  }
```

Comment:
These 6 all just worked for me:
```
028ec70462207b57e3d4d9332d9e0aee676c92d89b7c9fb0850fc2a24814d4d83c@71.171.123.161:9735
02eadbd9e7557375161df8b646776a547c5cbc2e95b3071ec81553f8ec2cea3b8c@18.118.231.3:9735
02312627fdf07fbdd7e5ddb136611bdde9b00d26821d14d94891395452f67af248@23.237.77.12:9735
02889be42fc32093d2dcbfa59369df262e3577b333d8a45e5859dcdd6a4139839a@[2a09:8280:1::42:a6f3]:9735
031c64a68e6d1b9e50711336d92b434c584ce668b2fae59ee688bd73713fee1569@186.154.207.228:9735
038863cf8ab91046230f561cd5b386cbff8309fa02e3f0c3ed161a3aeb64a643b9@203.132.94.196:9735
```

Maybe just strike the ones that don't work from the list and add the 6 above?

Reply:
Hopefully these are somewhat reliable! I guess we can always revisit in a few months.

Reply:
Thanks for the review guys, I will update today the PR

Do you think that we are able to have a small script that will ping the nodes and make sure they are up?

We could inject them inside the CI. IIRC @tsjk has a nice script that we can modify?

Reply:
@vincenzopalazzo great idea! Then we can be alerted when they are no longer reliable.